app/models/user.rb

```diff
@@ -0,0 +1,3 @@
+class User < ActiveRecord::Base
+  attr_accessor :password
+end
```

Comment:
Same for the `UserHelper` and this `User` model, I think this probably belong to the feature Angela is working on to use OAuth2 for authentication.